client/starwhale/core/runtime/model.py

```diff
@@ -86,17 +87,22 @@
 
 RUNTIME_API_VERSION = "1.1"
 
+_t_mixed_str_list = t.Union[t.List[str], str]
+_list: t.Callable[[_t_mixed_str_list], t.List[str]] = (
+    lambda _x: _x if isinstance(_x, (list, tuple)) else [_x]
+)
+
 
 class Environment:
     def __init__(
         self,
-        arch: str = SupportArch.X86_64,
+        arch: _t_mixed_str_list = "",
         os: str = SupportOS.UBUNTU,
         python: str = DEFAULT_PYTHON_VERSION,
         cuda: str = DEFAULT_CUDA_VERSION,
         **kw: t.Any,
     ) -> None:
-        self.arch = arch.lower()
+        self.arch = _list(arch)
```

Comment:
Does the `list()` function work well for this purpose?

Reply:
```python
In [1]: list("abcd")
Out[1]: ['a', 'b', 'c', 'd']
```